src/index.js

```diff
@@ -3,11 +3,14 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
+import axios from 'axios';
 
 if (process.env.NODE_ENV !== "development") {
   console.log = () => {};
 }
 
+axios.defaults.withCredentials = true;
```

Comment:
@lazycipher I think this should be only when user is authenticated.
Add this setAuthToken function in 
```
/utils/setAuthToken.js
axios.defaults.withCredentials = true;
```